release.json

```diff
@@ -1,5 +1,6 @@
 {
-  "mongodb-kubernetes-operator": "0.7.5",
+  "golang-builder-image": "golang:1.18.5",
```

Comment:
Actually, could we use `golang:1.18` here and never worry about the micro updates?

On one hand I know the build reproducibility argument but on the other hand, this is the fastest way to ensure we're compatible with the latest micro of the Golang. I think this outweighs the build reproducibility argument.

Reply:
Yes, I like the idea!